tests/server/security/test_model.py

```diff
@@ -11,42 +11,68 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+import uuid
+from datetime import datetime
 
 import pytest
 from argilla.server.errors import EntityNotFoundError
-from argilla.server.security.model import User
+from argilla.server.security.model import User, UserCreate
```

Comment:
You are not using `UserCreate` right?

```suggestion
from argilla.server.security.model import User
```

Reply:
I'm using it in tests related to the username, since is the model which validate this behavior